js/controller.js

```diff
@@ -2441,7 +2451,7 @@ OO.plugin('Html5Skin', function(OO, _, $, W) {
       this.renderSkin();
     },
 
-    onChangeClosedCaptionLanguage: function(event, language) {
+    onChangeClosedCaptionLanguage: function(event, language, params = {}) {
```

Comment:
Since you're modifying the function signature, can you add a jsdoc for this?

Reply:
Added